### ISSUE TITLE
graphics chapter

### ISSUE THREAD

**Issue by DaveBackus** [DaveBackus/Data_Bootcamp_Book#22]
Add example with two y axes...

**Comment by sglyon** [comment]
I usually don't like plots with 2 y-axis -- I'm a fan of two subplots on top of one another. 

Do you have an example in mind?

**Comment by DaveBackus** [comment]
No, just a question I got in class.  If we do this, we can add your
advice.

**Comment by DaveBackus** [comment]
A great example, either for this chapter or seaborn.  

 https://cepa.stanford.edu/seda/download?nid=1727&destination=node/1717
)

// Spencer

On April 29, 2016 at 10:58:50 PM, David Backus (david.backus@gmail.com) wrote:

Yup. We might be able to get the data, follow the paper link.
